packages/core/content-type-builder/admin/src/components/AttributeOptions/EmptyAttributes/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+import { LinkButton } from '@strapi/design-system/LinkButton';
+import { Icon } from '@strapi/design-system/Icon';
+import Plus from '@strapi/icons/Plus';
+import EmptyStateDocument from '@strapi/icons/EmptyDocuments';
+import { getTrad } from '../../../utils';
+
+const EmptyCard = styled(Box)`
+  background: ${({ theme }) =>
+    `linear-gradient(180deg, rgba(234, 234, 239, 0) 0%, ${theme.colors.neutral150} 100%)`};
+  opacity: 0.33;
+`;
+
+export const EmptyCardGrid = () => {
+  return (
+    <Flex wrap="wrap" gap={4}>
+      {Array(4)
+        .fill(null)
```

Comment:
This could be simplified to 
```suggestion
      {[...Array(4)]
```